recipes/socketIO-client-nexus/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "socketIO-client-nexus" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41e6aaaff981f6729690f472eb2c7a5d1dda07349077e34444527b9700b0aea5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.6
```

Comment:
Is there a reason why the python version is different in `host` and `run`? Other than this, the recipe LGTM.

Reply:
copy pasta, fixed